### DISCUSSION TITLE
custom rule to create

### DISCUSSION THREAD

**Discussion by apujayed** [pocketbase/pocketbase#3619] [Q&A]
is there any way to do this if referer will be not domain.com than reject the request?i face an issue that i added a create rule  `@request.auth.id !="" `but stil if i use axios and my api to post request it successfully accept the request..how to handle this?

**Comment by ganigeorgiev** [comment]
_See https://github.com/pocketbase/pocketbase/discussions/3620._

If the above doesn't help, please elaborate and/or provide an example with the desired outcome.
